### PR TITLE
[hotfix][typo]Fix typo in `UnsignedTypeConversionITCase`

### PR DESCRIPTION
**Pull request by V1ncentzzZ**
## What is the purpose of the change

*(For example: This pull request makes task deployment go through the blob server, rather than through RPC. That way we avoid re-transferring them on each deployment (during recovery).)*

## Brief change log

Fix typo in `UnsignedTypeConversionITCase`

## Verifying this change

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (100MB)*
  - *Extended integration test for recovery after master (JobManager) failure*
  - *Added test that validates that TaskInfo is transferred only once across recoveries*
  - *Manually verified the change by running a 4 node cluser with 2 JobManagers and 4 TaskManagers, a stateful streaming program, and killing one JobManager and two TaskManagers during the execution, verifying that recovery happens correctly.*

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (yes / no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (yes / no)
  - The serializers: (yes / no / don't know)
  - The runtime per-record code paths (performance sensitive): (yes / no / don't know)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn/Mesos, ZooKeeper: (yes / no / don't know)
  - The S3 file system connector: (yes / no / don't know)

## Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

**Comment by V1ncentzzZ** [comment]
cc @leonardBang

File: flink-connectors/flink-connector-jdbc/src/test/java/org/apache/flink/connector/jdbc/table/UnsignedTypeConversionITCase.java [hunk 1]
```diff
@@ -95,7 +95,7 @@ public static void prepareMariaDB() throws IllegalStateException {
                     }
                 }
             } catch (Exception e) {
-                logger.warn("Initialize DB fail caused by {}", e);
+                logger.warn("Initialize DB fail caused by ", e);
```
Comment (leonardBang) [inline flink-connectors/flink-connector-jdbc/src/test/java/org/apache/flink/connector/jdbc/table/UnsignedTypeConversionITCase.java:98]: this should work properly?

File: flink-connectors/flink-connector-jdbc/src/test/java/org/apache/flink/connector/jdbc/table/UnsignedTypeConversionITCase.java [hunk 2]
```diff
@@ -106,7 +106,7 @@ public static void prepareMariaDB() throws IllegalStateException {
         if (!initDbSuccess) {
             throw new IllegalStateException(
                     String.format(
-                            "Initialize MySQL database instance failed after {} attempts,"
+                            "Initialize MySQL database instance failed after %s attempts,"
```
Comment (leonardBang) [inline flink-connectors/flink-connector-jdbc/src/test/java/org/apache/flink/connector/jdbc/table/UnsignedTypeConversionITCase.java:109]: nice catch? how about %s -> %d

**Review by leonardBang — COMMENTED** [review]
Thank @V1ncentzzZ for the contribution , Left minor comment

File: flink-connectors/flink-connector-jdbc/src/test/java/org/apache/flink/connector/jdbc/table/UnsignedTypeConversionITCase.java (hunk 1 above)
Comment (V1ncentzzZ) [reply flink-connectors/flink-connector-jdbc/src/test/java/org/apache/flink/connector/jdbc/table/UnsignedTypeConversionITCase.java]: Sure, but the `{}` is an useless character.

**Review by leonardBang — APPROVED** [review]
LGTM, +1 once the build passed

**Review by wuchong — APPROVED** [review]
LGTM.

**Comment by wuchong** [comment]
@V1ncentzzZ  could you open a pull request for master too?

**Comment by V1ncentzzZ** [comment]
Thanks @wuchong @leonardBang , this [PR](https://github.com/apache/flink/pull/14876) for master.

**Pull request merged**